cli/pkg/release/integrate/integrate.go

```diff
@@ -0,0 +1,238 @@
+package integrate
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/wordpress-mobile/gbm-cli/pkg/console"
+	"github.com/wordpress-mobile/gbm-cli/pkg/gbm"
+	"github.com/wordpress-mobile/gbm-cli/pkg/gh"
+	"github.com/wordpress-mobile/gbm-cli/pkg/render"
+	"github.com/wordpress-mobile/gbm-cli/pkg/repo"
+	"github.com/wordpress-mobile/gbm-cli/pkg/shell"
+)
+
+type Integration interface {
```

Comment:
I updated the release integration handler to use interfaces to handle the different platforms.